huaweicloud/services/acceptance/gaussdb/data_source_huaweicloud_gaussdb_opengauss_instances_test.go

```diff
@@ -156,5 +165,5 @@ data "huaweicloud_gaussdb_opengauss_instances" "test" {
     huaweicloud_gaussdb_opengauss_instance.test,
   ]
 }
-`, common.TestBaseNetwork(rName), rName)
+`, testDataSourceAccOpenGaussInstance_base(rName), rName)
```

Comment:
I think we can use `testAccOpenGaussInstance_basic` and `testAccOpenGaussInstance_haModeCentralized` to create OpenGauss instance.

Reply:
done